Tools/DumpRenderTree/mac/ResourceLoadDelegate.mm

```diff
@@ -118,6 +119,14 @@ - (NSString *)_drt_descriptionSuitableForTestResult
 
 @implementation ResourceLoadDelegate
 
+@synthesize mainResourceURL;
+
+- (void)dealloc
+{
+    mainResourceURL = nil;
```

Comment:
Just realized that this will leak because DumpRenderTree is not compiled with ARC.  (Assigning to `nil` is optional without ARC, but makes the switch to ARC easier later as you just delete the `-release` call.)

```suggestion
    [mainResourceURL release];
    mainResourceURL = nil;
```